tests/test_read.py

```
@@ -0,0 +1,70 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import os
+import random
+import string
+from unittest import mock
+import pytest
+
+from azure.core.credentials import AzureSasCredential, AzureNamedKeyCredential
+from azure.identity import DefaultAzureCredential
+
+from azstoragetorch.io import BlobIO
+from azstoragetorch._client import AzStorageTorchBlobClient
+
+from azure.core.exceptions import ResourceExistsError
+from azure.storage.blob import BlobServiceClient
+
+
+
+@pytest.fixture
+def account_url():
+    account_name = os.environ.get('PYTORCH_STORAGE_ACCOUNT_NAME', None) or "pytorchstorageaccount"
```

For this fixture, instead of setting a default account name, I'm think we just raise an error. So the fixture could look something like this:
```python
@pytest.fixture
def account_url():
    account_name = os.environ.get("AZSTORAGETORCH_STORAGE_ACCOUNT_NAME")
    if account_name is None:
        raise ValueError(
            f'"AZSTORAGETORCH_STORAGE_ACCOUNT_NAME" environment variable must be set to run end to end tests.'
        )
    return f"https://{account_name}.blob.core.windows.net"
```
The reason that I'm suggesting it is that if individuals run these tests, they likely do not have this environment variable set and also likely do not have access to the fall back storage account. So, it is both safer and clearer by explicitly raising an exception asking the operator to set the environment variable as otherwise they will just get authorization errors in not being able to access that account